### PR TITLE
Remove clutter for cs2101

### PR DESCRIPTION
**Pull request by nananakx-x**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -291,17 +291,17 @@ Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS housing/HOUSING_TYPE availa
...
+| `PHONE_NUMBER`   | Phone number of the fosterer                                                                                                 | `93456778`, `89067547`                                      |
+| `EMAIL`          | Email of the fosterer                                                                                                        | `thomas718@gmail.com`, `kate@yahoo.com.sg`                  |
+| `ADDRESS`        | Address of the fosterer                                                                                                      | `Orchard road, Blk 8, #13-04`                               |
+| `HOUSING_TYPE`   | - Housing type of the fosterer<br/> - Case-sensitive<br/> - Can be HDB / Condo / Landed / nil                                | `HDB`, `Condo`, `Landed`, `nil`                             |
+| `AVAILABILITY`   | - Availability of the fosterer<br/> - Case-sensitive<br/> - Can be Available / NotAvailable / nil                            | `NotAvailable`, `Available`, `nil`                          |
```
Comment (butteredyakiimo) [inline docs/UserGuide.md:301]: do u think we can remove the part about "can be HDB..." and for availability too! i feel like it might be a repeat :D

Comment (butteredyakiimo) [reply docs/UserGuide.md:301]: same for the one in edit too!

File: docs/UserGuide.md [hunk 2]
```diff
@@ -520,18 +520,18 @@ Format: `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [housing/HOU
...
-| `TAG`            | Tag to be associated with the fosterer                                                                                                                                                                                                                                               | `experienced`, `urgent`                                     |
+| Parameter        | About                                                                                                                         | Example                                                     |
+|------------------|-------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------|
+| `INDEX`     | - The index of a fosterer displayed in the list obtained from a `list`/`find` command<br/> - Index must be a positive integer | `1`, `2`, `3`                                               |
```
Comment (H1410101) [inline docs/UserGuide.md]:
```suggestion
| `INDEX`          | - The index of a fosterer displayed in the list obtained from a `list`/`find` command<br/> - Index must be a positive integer | `1`, `2`, `3`                                               |
```
Minor formatting things

File: docs/UserGuide.md (hunk 1 above)
Comment (nananakx-x) [reply docs/UserGuide.md:301]: wait its not mentioned anywhere else tho...? I mean, it's not mentioned anywhere else at the top. And for edit, the table is the same for standardisation

File: docs/UserGuide.md (hunk 2 above)
Comment (nananakx-x) [reply docs/UserGuide.md]: done!

**Review by butteredyakiimo — APPROVED** [review]

**Pull request merged**